misk/src/test/kotlin/misk/web/metadata/ServiceMetadataActionTest.kt

```diff
@@ -0,0 +1,62 @@
+package misk.web.metadata
+
+import com.squareup.moshi.Moshi
+import misk.client.HttpClientEndpointConfig
+import misk.client.HttpClientFactory
+import misk.moshi.adapter
+import misk.security.authz.FakeCallerAuthenticator
+import misk.testing.MiskTest
+import misk.testing.MiskTestModule
+import misk.web.jetty.JettyService
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import javax.inject.Inject
+
+@MiskTest(startService = true)
+class ServiceMetadataActionTest {
```

Comment:
The body of this test is shamelessly copied from here: https://github.com/cashapp/misk/blob/fe47b15b4ac8734e810d74dc2ff1cf6fa90580bf/misk/src/test/kotlin/misk/web/metadata/DashboardMetadataActionTest.kt#L24

Reply:
Surprised there wasn't already one, would have caught this regression!